style.css

```diff
@@ -0,0 +1,6 @@
+/*I'm thinking a background color of some kind of gradient....something dark though...
```

Comment:
I wouldn't worry too much about the styles yet. Let's get the functional code working, and then you can always style it up the way you want. 

Megan has a great gradient tool that she was showing me yesterday. Ask her about it, and I'm sure she'll be happy to share it with you!